compiler/rustc_passes/src/reachable.rs

```diff
@@ -266,6 +277,27 @@ impl<'tcx> ReachableContext<'tcx> {
             }
         }
     }
+
+    /// Finds anonymous nested statics created for nested allocations.
```

Comment:
What does "find" mean here? This function seems to be doing absolutely nothing -- it recurses the static value to find nested statics, and then doesn't do anything with them.